package.json

```diff
@@ -15,7 +15,7 @@
     "dev-web": "npm run dev -w web-local",
     "dev-runtime": "go run cli/main.go start dev-project --no-ui",
     "clean": "rm -rf dev-project",
-    "test": "npm run test -w web-common && npm run test -w web-local && npm run test -w web-auth"
+    "test": "npm run test -w web-common & npm run test -w web-auth & make cli && npm run test -w web-local"
```

Comment:
qq: typo at single "&" at `web-common & npm run test`?

Reply:
The intention is to run `npm run test -w web-common`,  `npm run test -w web-auth` and `make cli` concurrently since they aren't dependent on each other. After that completes, `npm run test -w web-local` will run.

The two unit tests are pretty fast, but I just want to start `make cli` as soon as possible since it takes the longest.

Reply:
gotcha, makes sense, great idea!